docs/source/llms/deployments/uc_integration.rst

```diff
@@ -0,0 +1,164 @@
+Unity Catalog Integration
+-------------------------
+
+This example demonstrates how to use the Unity Catalog (UC) integration with MLflow Deployments server.
+
+Pre-requisites
+--------------
+
+1. Clone the MLflow repository:
+
+.. code-block:: bash
+
+    # Clone the MLflow repository
+    git clone https://github.com/mlflow/mlflow.git
+    cd mlflow
+
+2. Install the required packages:
+
+.. code-block:: bash
+
+    # Install the required packages
+    pip install mlflow openai databricks-sdk
+
+3. Create the UC function used in the example script on your Databricks workspace by running the following SQL command:
+
+.. code-block:: sql
+
+    CREATE OR REPLACE FUNCTION
+    my.uc_func.add (
+      x INTEGER COMMENT 'The first number to add.',
+      y INTEGER COMMENT 'The second number to add.'
+    )
+    RETURNS INTEGER
+    LANGUAGE SQL
+    RETURN x + y
+
+
+To define your own function, see https://docs.databricks.com/en/sql/language-manual/sql-ref-syntax-ddl-create-sql-function.html#create-function-sql-and-python.
+
+4. Create a SQL warehouse by following the instructions at https://docs.databricks.com/en/compute/sql-warehouse/create.html.
+
+Running the deployments server
+------------------------------
+
+Once you have completed the pre-requisites, you can start the deployments server:
+
+.. code-block:: bash
+
+    # Required to authenticate with Databricks. See https://docs.databricks.com/en/dev-tools/auth/index.html#supported-authentication-types-by-databricks-tool-or-sdk for other authentication methods.
+    export DATABRICKS_HOST="..."
+    export DATABRICKS_TOKEN="..."
+
+    # Required to execute UC functions. See https://docs.databricks.com/en/integrations/compute-details.html#get-connection-details-for-a-databricks-compute-resource for how to get the http path of your warehouse.
+    # The last part of the http path is the warehouse ID.
+    #
+    # /sql/1.0/warehouses/1234567890123456
+    #                     ^^^^^^^^^^^^^^^^
+    export DATABRICKS_WAREHOUSE_ID="..."
+
+    # Enable Unity Catalog integration
+    export MLFLOW_ENABLE_UC_FUNCTIONS=true
+
+    # Run the deployments server
+    mlflow deployments start-server --config-path examples/deployments/deployments_server/openai/config.yaml --port 7000
+
+Running the example script
```

Comment:
```suggestion
Query the Endpoint with UC Function
```
nit: to explain what this step is for (there's no description for what this example script does)

Reply:
sg!